content/crutchcorn/posts/advice-for-new-twitch-streamers/index.md

```diff
@@ -3,7 +3,6 @@
     title: "Advice for New Twitch Streamers",
     description: "Twitch streaming can be fun and fulfilling, but has various roadblocks in the way for newcomers. Here's how you can overcome them and grow your Twitch channel.",
     published: '2023-03-25T21:52:59.284Z',
-    authors: ['crutchcorn'],
```

Comment:
Was this removal intentional?

Reply:
Correction: It probably is, given the goals of this content folder restructure. Why don't we remove this property from every article?

Reply:
This is meant to be an optional property, yes. The way that multiple authors are specified would be by listing all authors in this array, but it defaults to the author it's nested under when omitted.